test_runner/regress/test_ondemand_download.py

```diff
@@ -465,14 +463,11 @@ def get_resident_physical_size():
         assert query_scalar(cur, "select count(*) from testtab") == table_len
 
 
-@pytest.mark.parametrize("remote_storage_kind", [RemoteStorageKind.MOCK_S3])
-def test_compaction_downloads_on_demand_without_image_creation(
-    neon_env_builder: NeonEnvBuilder, remote_storage_kind: RemoteStorageKind
-):
+def test_compaction_downloads_on_demand_without_image_creation(neon_env_builder: NeonEnvBuilder):
```

Comment:
This test used MOCK_S3, not LOCAL_FS. The PR description only mentions cleanup for single-parametrized LOCAL_FS tests.
Please address one way or the other, also see other tests.

Reply:
I think you misread? This used be parametrized for `[RemoteStorageKind.MOCK_S3]`, is now using `MOCK_S3`.

Checked both 76079589c, 8193d4900, they look good.